src/client/app/admin/authentication/authentication.service.ts

```diff
@@ -50,7 +51,7 @@ export class AuthenticationService {
 	};
 
 	// Get the user who is currently logged in (or null if no one is logged in)
-	public getCurrentUser(): User {
+	public getCurrentUser() {
```

Comment:
Authentication service should be able to depend on user. So, it'd be better to leave the return type here and refactor the code to allow the dependency. We shouldn't be making the code less correct as part of a convenience refactor.